rules/resources.bzl

```diff
@@ -381,7 +381,7 @@ OUT_DIR=$(mktemp -d)
 CUR_PWD=$(pwd)
 
 if zipinfo -t "$1"; then
-    ORDERED_LIST=`(unzip -l "$1" | sed -e '1,3d' | head -n -2 | tr -s " " | cut -d " " -f5)`
+    ORDERED_LIST=`(zipinfo -1 "$1" | sort)`
```

Comment:
This relies on zipinfo being installed on every executor machine. I'm not sure if that's the case for the Google RBE machines (zip and unzip are certainly installed). Is this a crucial change?

Reply:
`zipinfo` should already be installed given that the previous line invokes it. If it weren't, this would fail before getting here.

_Some_ change needs to be made to this line, because `head -n -2` isn't portable. `head -n -2` could be replaced by `tail -r | tail -n +3 | tail -r`, but even that isn't strictly portable. It does appear to be more portable than `head -n -2` though.

But understanding the purpose of this line, `zipinfo -1` makes the most sense. And because it's used in the previous line, I think it's safe.